crate/yew_styles/src/components/button.rs

```diff
@@ -44,7 +44,7 @@ use yew::{utils, App};
 ///         match msg {
 ///             Msg::Clicked(greeting) => {
 ///                 let mut console = ConsoleService::new();
-///                 console.log(format!("{}", greeting))
+///                 console.log(format!("{}", greeting).as_str())
```

Comment:
good catch however I think is better include '&' instead of `.as_str()` that means that you pass a reference
```
console.log(&format!("{}", greeting));
```

Reply:
Great suggestion! Have made the change